utils/table_to_annotations.py

```diff
@@ -0,0 +1,277 @@
+    print(f"Annotations acquired from Synapse table {source_id}")
+
+    return annotations_table
+
+
+def collect_fileview_annotations(syn, files: list, fileview_id: str) -> dict:
+    """Collect all entries from a File View metadata table,
+    extract Biospecimen and File identifiers,
+    return a Biospecimen Key: File Synapse Id dictionary"""
+    
+    id_key_tuples = []
+
+    fileview_table = get_annotations_table(syn, fileview_id)
+
+    for _, row in fileview_table.iterrows():
+        id_key_tuple = (row["Biospecimen Key"], row["FileView_id"])
+        id_key_tuples.append(id_key_tuple)
+    
+    file_tuples = dict([tup for tup in id_key_tuples if tup[1] in files])
+
+    return file_tuples
+            
+
+def collect_biospecimen_annotations(syn, file_tuples: dict, specimen_info_tuple: tuple[str, str, str]) -> tuple[dict, dict]:
+    """Collect all entries from a Biospecimen metadata table,
+    select entries where Biospecimen_id is present in Biospecimen Keys associated with files,
+    match Biospecimen metadata with column names to create key:value pairs,
+    apply annotations to each file,
+    return Individual Key:File Id and Model Key: File Id dictionaries
+    """
+    
+    individual_tup_list = []
+    model_tup_list = []
+    
+    component, table_id, column_list = specimen_info_tuple
+    data_table = get_annotations_table(syn, table_id)
+    count = 0
+    for _, row in data_table.iterrows():
+        id = row["Biospecimen_id"]
+        if id in file_tuples.keys():
+            file_id = file_tuples[id]
+            individual_id = row["Individual Key"]
+            model_id = row["Model Key"]
+            individual_tup = (individual_id, file_id)
+            individual_tup_list.append(individual_tup)
+            model_tup = (model_id, file_id)
+            model_tup_list.append(model_tup)
+            biospecimen_annotations = list(zip(column_list, list(row)))
+            biospecimen_annotations = apply_annotations_to_entity(syn, component, file_id, biospecimen_annotations)
+            count += 1
+    
+    print(f"Biospecimen annotations applied to {count} entities")
+    return dict(individual_tup_list), dict(model_tup_list)
+
+
+def collect_record_annotations(syn, info_tuple: tuple[str, str, str], tuple_dict: dict):
+    """Collect all entries from a Synapse table,
+    select entries where primary key (e.g. Individual_id) matches foreign key (e.g. Individual Key),
+    based on dictionary output from collect_biospecimen_annotations,
+    apply annotations to each file"""
+    
+    component, table_id, column_list = info_tuple
+    data_table = get_annotations_table(syn, table_id)
+    count = 0
+    for _, row in data_table.iterrows():
+        id = row[f"{component}_id"]
+        if id in tuple_dict.keys():
+            file_id = tuple_dict[id]
+            annotations = list(zip(column_list, list(row)))
+            annotations = apply_annotations_to_entity(syn, component, file_id, annotations)
```

Comment:
Stale variable, as it's not used after this. 

Additionally, since `apply_annotations_to_entity()` doesn't return anything, assigning its result to a variable doesn't really serve a purpose and so, could be removed:

```suggestion
            apply_annotations_to_entity(syn, component, file_id, annotations)
```

Reply:
Got it! I found another stale variable in `apply_annotations_to_entity()` that I went ahead and removed, too.